File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/ILocalizationResourceContributor.cs
using System.Collections.Generic;
using Microsoft.Extensions.Localization;

namespace Volo.Abp.Localization;

public interface ILocalizationResourceContributor
{
    void Initialize(LocalizationResourceInitializationContext context);

    LocalizedString GetOrNull(String cultureName, String name);

    void Fill(String cultureName, Dictionary<String, LocalizedString> dictionary);
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/ILocalizationResourceContributor.java
package com.ueueo.localization;

import java.util.Map;

public interface ILocalizationResourceContributor
{
    void initialize(LocalizationResourceInitializationContext context);

    LocalizedString getOrNull(String cultureName, String name);

    void fill(String cultureName, Map<String, LocalizedString> dictionary);
}
